web/src/app/schedules/ScheduleTZFilter.js

```diff
@@ -41,7 +41,13 @@ export function ScheduleTZFilter(props) {
       control={
         <Switch
           checked={zone !== 'local'}
-          onChange={e => setZone(e.target.checked ? tz : 'local')}
+          onChange={e => {
+            const newZone = e.target.checked ? tz : 'local'
+            setZone(newZone)
+            if (typeof props.onChange === 'function') {
+              return props.onChange(newZone)
```

Comment:
Shouldn't need this, we should always be reading from the `tz` prop

Reply:
This change will be dependent on the [above conversation](https://github.com/target/goalert/pull/261#discussion_r357352326) ^